posydon/grids/termination_flags.py

```diff
@@ -231,8 +232,52 @@ def infer_interpolation_class(tf1, tf2):
     if tf2 in TF2_POOL_NO_RLO:
         return "no_MT"
     if tf1 in TF1_POOL_STABLE:
+        if (binary_history is not None and history1 is not None):
+            rl_overflow1 = binary_history['rl_relative_overflow_1']
+            rl_overflow2 = binary_history['rl_relative_overflow_2']
+            center_h1 = history1['center_h1']
+            forward = np.logical_and(rl_overflow1>RL_RELATIVE_OVERFLOW_THRESHOLD,
+                                     rl_overflow2<RL_RELATIVE_OVERFLOW_THRESHOLD)
+            reverse = np.logical_and(rl_overflow1<RL_RELATIVE_OVERFLOW_THRESHOLD,
+                                     rl_overflow2>RL_RELATIVE_OVERFLOW_THRESHOLD)
+            reverse_postMS = np.logical_and(reverse,
+                                           center_h1<THRESHOLD_CENTRAL_ABUNDANCE)
+            if (np.max(forward) == True and np.max(reverse_postMS) == True):
```

Comment:
Why do you request to be post MS?

Reply:
I want to exclude the ones with very close orbits where they have MT from both stars when they are MS

Reply:
In which way are they different, that they need to be excluded? Do we have many of forward and backward MT on MS, which is not going into a contact system?

Reply:
The reason we separate reverse_MT is that the MT would cause spin up of the star_1 and star_2 would be a partially stripped postMS star, which are very different from forward_MT cases.  If the reverse_MT happaned when the star1 is still in MS, the binary is more like the forward_MT case.

Reply:
The spin is only one reason, the masses are different as well, because with reverse MT star1 gained mass, while without reverse MT star 1 never gained mass. This holds for the MS case, too.
I don't really get, why does the reverse MT on MS don't lead to critical rotation like you are suggesting?

Reply:
The mass gained through reverse MT should be little because of rotation. And MS stars will lose AM after because of forward MT and winds so it will end with a low spin anyway. While for the reverse MT cases I am talking about here, the primary star is spun up close to SN and it will end with a moderate spin.

Reply:
OK, for initial-final interpolation, you are correct.
But we need to be aware, when we would like to get interpolation during the evolution in the future, this will cause trouble there, without anybody remembering this to be the reason.